blockchain/chain_sync/Cargo.toml

```diff
@@ -65,31 +65,31 @@ forest_json_utils      = "0.1"
 forest_message         = { version = "0.7", features = ["proofs", "blst"] }
 forest_test_utils      = { version = "0.1.0", features = ["test_constructors"] }
 forest_vm              = "0.3.1"
-futures                = "0.3.5"
-futures-util           = "0.3.5"
-fvm                    = "1.0"
+futures                = "0.3.24"
+futures-util           = "0.3.24"
+fvm                    = "1.1"
 fvm_ipld_car           = "0.4.1"
 fvm_ipld_encoding      = "0.2"
 fvm_shared             = { version = "0.8.0", default-features = false }
 hex                    = "0.4"
 lazy_static            = "1.4.0"
-libp2p                 = { version = "0.40.0-rc.1", default-features = false }
-log                    = "0.4.8"
-lru                    = "0.7.2"
+libp2p                 = { version = "0.40.0", default-features = false }
```

Comment:
@LesnyRumcajs Is it a problem that this `libp2p` dependency is still `0.40` rather than `0.46` like in the rest of Forest?

Reply:
I am not sure why this dependency was really introduced [here](https://github.com/ChainSafe/forest/commit/d2275cdcad86091f9d26240cb02e7dd8a559197a#diff-9c1028edc4d88e02c0d5a5f32a7426794248e9c57d1b613671f7abc910861d1e), as I reviewed all occurrences when bumping `libp2p`. In general something strange happened to dev-dependencies there, maybe a conflict fiasco?

Anyway, it should not be using `0.40.0` but `0.46`. Most likely it should be deleted if possible.

Reply:
Yeah, conflict mishap. :(

@tyshko5 Could you verify that all the dev-dependencies for `chain_sync` are correct?

Reply:
Ok